test/scala/com/github/johnynek/bazel_deps/ParseTest.scala

```diff
@@ -25,6 +25,7 @@ class ParseTest extends FunSuite {
             Map(ArtifactOrProject("scalding") ->
               ProjectRecord(
                 Language.Scala.default,
```

Comment:
can you add some hand written tests that include classifiers?

Reply:
Will do.